apps/web/lib/app-providers-app-dir.tsx

```diff
@@ -26,10 +25,11 @@ import { getThemeProviderProps } from "./getThemeProviderProps";
 export type AppProps = Omit<
   NextAppProps<
     WithLocaleProps<
-      WithNonceProps<{
```

Comment:
Avoiding use of _WithNonceProps_, it was just adding nonce property, which we can add ourselves.